### PR TITLE
[IMP] enable the unreserve stock move on a Manufacturing order

### PR DESCRIPTION
**Pull request by nseinlet**
@jco-odoo 
Could you have a look at this?

### CODE REVIEW

File: addons/mrp/mrp.py [hunk 1]
```diff
@@ -1016,6 +1016,17 @@ def action_produce(self, cr, uid, production_id, production_qty, production_mode
         self.signal_workflow(cr, uid, [production_id], 'button_produce_done')
         return True
 
+    def button_unreserve(self, cr, uid,  production_ids, context=None):
+        #import pudb
+        #pudb.set_trace()
+        productions = self.read(cr, uid, production_ids, ['move_lines', 'move_created_ids'], context=context)
+        self.pool.get('stock.move').action_cancel(cr, uid, productions[0]['move_lines'], context=context)
```
Comment (jco-odoo) [inline addons/mrp/mrp.py]: Normally, unreserving should be sufficient, why cancelling and deletion?

Comment (nseinlet) [inline addons/mrp/mrp.py]: moves seems to be created twice, so I've canceled them before they were recreated.

File: addons/mrp/mrp.py [hunk 2]
```diff
@@ -1016,6 +1016,17 @@ def action_produce(self, cr, uid, production_id, production_qty, production_mode
         self.signal_workflow(cr, uid, [production_id], 'button_produce_done')
         return True
 
+    def button_unreserve(self, cr, uid,  production_ids, context=None):
+        #import pudb
```
Comment (sve-odoo) [inline addons/mrp/mrp.py]: Are you expecting a lot of bugs here? :laughing:

File: addons/mrp/mrp.py (hunk 1 above)
Comment (jco-odoo) [inline addons/mrp/mrp.py]: hm, indeed, but you will create a lot of problems when these moves where MTO.   I would prefer to rethink it for master than to do bad things here.

**Comment by C3POdoo** [comment]
Hello,

This is an automatic message.

Thank you for your contribution but the version 8.0 is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.

If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards,

**Pull request closed without merging**